index.html

```diff
@@ -10,185 +10,225 @@
-
-            <div class="get">Get my resume</div>
-            <ul>
-                <li>
-                    <div class="lang-dropper">Languages  <img src="./images/dropBtn.png" id="dropBtn1" alt=""></div>
-                </li>
-            </ul>
-            <div class="lang"><img src="images/js.png" alt="">javaScript</div>
-            <div class="lang"><img src="images/html.png" alt="">HTML</div>
-            <div class="lang"><img src="images/CSS.png" alt="">CSS</div>
-            <ul class="footer-list">
-                <li></li>
-                <div class="frame">Frameworks <img src="./images/dropBtn2.png" id="dropBtn2" alt=""></div>
-                </li>
-                <li>
-                    <div class="frame"><span id="span">Skills</span><img src="./images/dropBtn2.png" id="dropBtn2" alt=""></div>
-                </li>
-            </ul>
         </section>
 
         <footer class="Contact-form">
-                <div class="contact-title">Contact me</div>
-                <div class="contact-content">
-                    If you have an application you <br> are interested in 
-                    developing, a <br> feature that you 
-                    need built or a <br> project that needs coding. I’d <br> love to help with it 
-                </div>
-                <form action="https://formspree.io/f/xvolalez" method="post">
-                    <ul>
-                        <li><input type="text" name="for-name" id="name" placeholder="Put your name" maxlength="30" required></li>
-                        <li><input type="email" name="for-mail" id="mail" placeholder="put your adress email" required></li>
-                        <li><textarea name="for-text" id="textarea" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required></textarea></li>
-                        <li><button>Get it Touch</button></li>
-                    </ul>            
-                </form>
+            <div class="contact-title">Contact me</div>
+            <div class="contact-content">
+                If you have an application you  are interested in
+                developing, a feature that you
+                need built or a  project that needs coding. I’d  love to help with it
+            </div>
+            <form action="https://formspree.io/f/xvolalez" method="post">
+                <ul>
+                    <li><input type="text" name="for-name" id="name" placeholder="Put your name" maxlength="30"
+                            required></li>
+                    <li><input type="email" name="for-mail" id="mail" placeholder="put your adress email" required>
+                    </li>
+                    <li><textarea name="for-text" id="textarea" maxlength="500" cols="30" rows="10"
+                            placeholder="Write your message here" required></textarea></li>
+                    <li><button>Get it Touch</button></li>
+                </ul>
+            </form>
         </footer>
```

Comment:
Kindly try to match the `background-image` for the `footer` section with the Figma design in the desktop version.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155524658-168a5d40-9000-4be7-bea7-e5e357048e7f.png)) | ![Template 2-Main (2) - Copy](https://user-images.githubusercontent.com/24703171/155524534-978396ec-61cb-4270-8c5b-3180a490c731.png) |